templates/blocks/header.twig

```diff
@@ -1,21 +1,25 @@
 <div class="page-header col-md-8 col-lg-8 col-sm-12">
-    <div class="top-tags-wrap">
-      {% if ( page_tags ) %}
-        {% for page_tag in page_tags %}
-          <span>#{{ page_tag.name }}</span>
-        {% endfor %}
-      {% endif %}
-    </div>
-    {% if ( header_title ) %}
-        <h1>{{ header_title|e('wp_kses_post')|raw }}</h1>
-    {% endif %}
-    {% if ( header_subtitle ) %}
-      {{ header_subtitle }} <BR><BR>
-    {% endif %}
-    {% if ( header_description ) %}
-       {{ header_description }} <BR><BR>
-    {% endif %}
-    {% if ( header_button_title ) %}
-       <a class="btn btn-danger" href="{{ header_button_link }}">{{ header_button_title }}</a>
-    {% endif %}
-</div>
\ No newline at end of file
+	<div class="top-tags-wrap">
+		{% if ( page_tags ) %}
+			{% for page_tag in page_tags %}
+				<span>#{{ page_tag.name }}</span>
+			{% endfor %}
+		{% endif %}
+	</div>
+	{% if ( header_title ) %}
+		<h1>{{ header_title|raw }}</h1>
```

Comment:
What's the reason for removing the `wp_kses_post` filtering?

Reply:
We are using autoescape so we won't need `wp_kses_post ` here. Also I need to remove `raw` as well.
- If we used `raw` then we need `wp_kses_post `